### PR TITLE
feat(demo): demo on pr

### PR DESCRIPTION
**Pull request by viv9k**
Signed-off-by: vivek kumar <vivekkumar@viveks-MacBook-Air-2.local>

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -21,6 +21,7 @@
 </head>
 
 <body>
+    Hello
```
Comment (Shrishti1802) [inline public/index.html:24]: **remove hello write welcome**

**Review by Shrishti1802 — CHANGES_REQUESTED** [review]
plz make changes

File: public/index.html (hunk 1 above)
Comment (viv9k) [reply public/index.html]: done

**Review by Shrishti1802 — APPROVED** [review]
ship it

**Comment by Shrishti1802** [comment]
demo completed

**Pull request closed without merging**